Makefile

```diff
@@ -26,6 +33,58 @@ interactive_manager:
 push:
 	$(MAKE) -C $(WORKING_DIR)/k8s/ push
 
+.PHONY: install
+install: gle client coordinator gae gie
+
+.PHONY: client
+client:
```

Comment:
The target `client` should depends on `gle`.

Reply:
Modified
```shell
.PHONY: client
client: gle

.PHONY: coordinator
coordinator: client
```